types/src/v0/v0_3/stake_table.rs

```diff
@@ -44,7 +47,38 @@ pub struct Validator<KEY: SignatureKey> {
     // commission
     // TODO: MA commission is only valid from 0 to 10_000. Add newtype to enforce this.
     pub commission: u16,
-    pub delegators: BTreeMap<Address, U256>,
+    pub delegators: HashMap<Address, U256>,
+}
+
+pub(crate) fn to_fixed_bytes(value: U256) -> [u8; std::mem::size_of::<U256>()] {
+    let bytes: [u8; std::mem::size_of::<U256>()] = value.to_le_bytes();
+    bytes
+}
+
+impl<KEY: SignatureKey> Committable for Validator<KEY> {
+    fn commit(&self) -> Commitment<Self> {
+        let mut schnorr_key_bytes = vec![];
+        self.state_ver_key.serialize_with_mode(&mut schnorr_key_bytes, ark_serialize::Compress::Yes).unwrap();
+
+        let mut builder = RawCommitmentBuilder::new(&Self::tag())
+            .fixed_size_field("account", &self.account)
+            .var_size_field("stake_table_key", self.stake_table_key.to_bytes().as_slice())
+            .var_size_field("state_ver_key", &schnorr_key_bytes)
+            .fixed_size_field("stake", &to_fixed_bytes(self.stake))
+            .u16(self.commission);
```

Comment:
nit: I think we usually use a domain separator, even if it's sometimes not strictly required.